lib/plugins/authldap/auth.php

```diff
@@ -201,7 +201,7 @@ protected function _getUserData($user, $inbind = false) {
 
         $this->_debug('LDAP user search: '.htmlspecialchars(ldap_error($this->con)), 0, __LINE__, __FILE__);
         $this->_debug('LDAP search at: '.htmlspecialchars($base.' '.$filter), 0, __LINE__, __FILE__);
-		$sr     = $this->_ldapsearch($this->con, $base, $filter, $this->getConf('userscope'));
+		$sr     = $this->_ldapsearch($this->con, $base, $filter, $this->getConf('userscope'), $this->getConf('attributes'));
 
```

Comment:
At line 265 is another _ldapsearch(). I no idea how this is actually working, but it is intentionally that at line 265 no attributes config setting is used?

Reply:
:-( That is a group query while this is a user query. I guess `userattributes` is a more accurate name here?

However I am not sure how group query needs additional attributes...